containers/PriceFeed.ts

```diff
@@ -9,7 +9,9 @@ import Token from "./Token";
 function usePriceFeed() {
   const { symbol: tokenSymbol } = Token.useContainer();
 
-  const [latestPrice, setLatestPrice] = useState<BigNumber | null>(null);
+  const [latestPrice, setLatestPrice] = useState<BigNumberish | undefined>(
+    undefined
+  );
```

Comment:
curious why `undefined` is used here instead of `null`

Reply:
No reason, I'll change to `null`. Curious if you prefer one over the other?

FWIW the `source` has to be undefined because Link's `href` prop expects `string | undefined`

Reply:
`null` tends to be a bit more intentional because a lot of things can be `undefined` "naturally" (e.g. referencing a non-existent property in an object), but `null` is usually set by a programmer manually. That being said, it doesn't _really_ matter that much because `null` has other problematic behaviours as well, there really is no right answer here.

A lot of discussion on the web, like this: https://stackoverflow.com/questions/5076944/what-is-the-difference-between-null-and-undefined-in-javascript